### PR TITLE
Style changes with autoformatter

### PR DESCRIPTION
**Pull request by NishanthJKumar**
This PR introduces fairly sweeping stylistic changes that are in line with the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guide. More usefully, these changes can be automatically made by included autoformatters, making it easy for developers to adhere to this style. Some highlights of changes:

* The ending triple-quote of a docstring (""") is now uniformly on the same line as the end of the docstring
* All arithmetic operators have one space before and after them (so a+b is now a + b everywhere)
* In-line comments occur 2 spaces away from the end of code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +39,9 @@ A simple implementation of search-then-sample task and motion planning is provid
...
 * The static typing check uses Mypy to verify type annotations. To run locally: `mypy . --config-file mypy.ini`. If this doesn't work due to import errors, try `mypy -p predicators --config-file predicators/mypy.ini` from one directory up.
 * The linter check runs pylint with the custom config file `.predicators_pylintrc` in the root of this repository. Feel free to edit this file as necessary. To run locally: `pytest . --pylint -m pylint --pylint-rcfile=.predicators_pylintrc`.
-* In addition to the packages in `requirements.txt`, please `pip install` the following packages if you want to contribute to the repository: `pytest-cov>=2.12.1` and `pytest-pylint>=0.18.0`. Also, install `mypy` from source: `pip install -U git+git://github.com/python/mypy.git@9a10967fdaa2ac077383b9eccded42829479ef31`. (Note: if [this mypy issue](https://github.com/python/mypy/issues/5485) gets resolved, we can install from head again.)
+* The autoformatting check uses the custom `.style.yapf` in this repo and checks for both . You can run the autoformatter locally with `yapf -i -r --style .style.yapf . && docformatter -i -r .`
```
Comment (ronuchit) [inline README.md:46]: Add period to end of line (for grammar).

File: .style.yapf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[style]
+based_on_style = pep8
+split_before_logical_operator = true
+column_limit = 78
```
[the remark this answers is not in the document]
Comment (NishanthJKumar) [reply .style.yapf:4]: it helped, but didn't completely resolve

[the remark this answers is not in the document]
Comment (NishanthJKumar) [reply .style.yapf:4]: i think there's not much difference between 79 and 78, I was just playing around. Will set it back to 79

**Review by ronuchit — APPROVED** [review]
woohoo!!

File: .github/workflows/predicators.yml [hunk 3]
```diff
@@ -59,4 +59,15 @@ jobs:
...
\ No newline at end of file
+        pytest . --pylint -m pylint --pylint-rcfile=.predicators_pylintrc
+  autoformat:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: run YAPF to test if python code is correctly formatted
```
Comment (ronuchit) [inline .github/workflows/predicators.yml:70]: sorry actually let's capitalize "Run" here :)

**Pull request merged**